### PR TITLE
Fix some brittle snapshots based on github api responses

### PR DESCRIPTION
**Pull request by ZoteTheMighty**
Updates snapshot tests, should unblock #71

### CODE REVIEW

**Comment by ZoteTheMighty** [comment]
@matthargett I can't formally request you as a reviewer, but merging this and updating #71 should unblock its CI so we can get it in.

**Review by matthargett — APPROVED** [review]

**Comment by matthargett** [comment]
> @matthargett I can't formally request you as a reviewer, but merging this and updating #71 should unblock its CI so we can get it in.

I approved, but it looks like is still needs another review somehow?

**Review by matthargett — APPROVED** [review]
:shipit:

**Pull request merged**
